### PR TITLE
Update contributor guidelines

### PR DESCRIPTION
**Pull request by findingsimple**
### Changes proposed in this Pull Request:

Updates contributor guidelines, pointing to Wiki articles we can tweak/maintain as needed.

Preparing for contributions from outside of the Woo Ventures team so that we have a common understanding (based on some of our internal docs)

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -6,6 +6,21 @@ Thanks for your interest in contributing to Google Listings and Ads!
 
 Like the WooCommerce project, we want to ensure a welcoming environment for everyone. With that in mind, all contributors are expected to follow our [Code of Conduct](./CODE_OF_CONDUCT.md).
 
+If you wish to contribute code, please read the information in the sections below. Then [fork](https://help.github.com/articles/fork-a-repo/) Google Listings & Ads, commit your changes, and [submit a pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) 🎉
+
+Google Listings & Ads is licensed under the GPLv3+, and all contributions to the project will be released under the same license. You maintain copyright over any contribution you make, and by submitting a pull request, you are agreeing to release that contribution under the GPLv3+ license.
```
Comment (ecgan) [inline .github/CONTRIBUTING.md:11]: Are we sure about GPLv3+, or should it be GPLv3? 🤔

**Review by ecgan — APPROVED** [review]
LGTM, and tested all the links and they are working as expected. Left one minor comment about license, just to be sure.

Note that I have also fixed formatting in [`bfc22dc` (#967)](https://github.com/woocommerce/google-listings-and-ads/pull/967/commits/bfc22dcaff5a85add8824012e348c501126196fa).

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (ecgan) [reply .github/CONTRIBUTING.md:11]: Just notice that this is similar to [WooCommerce Core](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md), so I guess we are good.

**Comment by findingsimple** [comment]
Thanks @ecgan !

I'm not sure what the changes did (apart from being a style preference) but they work for me 👍

Screen Shot 2021-08-20 at 10 21 43 am

Reference - [Github flavored markdown](https://github.github.com/gfm/#list-items)

**Comment by ecgan** [comment]
@jconroy , if you click on the source diff button, then you'll see the difference. See image below. It's basically just markdown formatting change based on [WordPress prettier config](https://www.npmjs.com/package/@wordpress/prettier-config).

image

**Comment by findingsimple** [comment]
@ecgan  thanks I get that - I was pointing out that there is no visible difference and its valid GIthub Markdown. I've never actually see that formatting before with 4? spaces between the list identifier and the the content 🤔  Its kinda unusual syntax

**Comment by ecgan** [comment]
> I've never actually see that formatting before with 4? spaces between the list identifier and the the content 🤔 Its kinda unusual syntax

@jconroy, yeah, I was puzzled too at first. Apparently that's how [`@wordpress/prettier-config`](https://www.npmjs.com/package/@wordpress/prettier-config) is configured, and we can see the same four spaces in https://raw.githubusercontent.com/WordPress/gutenberg/trunk/README.md and https://raw.githubusercontent.com/woocommerce/woocommerce/trunk/SECURITY.md. 

I think it may be related to this Prettier issue: Tab width should not affect text alignment after the bullet points in Markdown lists [#5019](https://github.com/prettier/prettier/issues/5019).

**Pull request merged**